packages/api/src/middlewares/body.ts

```diff
@@ -1,100 +1,50 @@
 import type { PieceContext } from '@sapphire/pieces';
-import { URLSearchParams } from 'url';
-import { createGunzip, createInflate } from 'zlib';
 import type { ApiRequest } from '../lib/structures/api/ApiRequest';
 import type { ApiResponse } from '../lib/structures/api/ApiResponse';
 import { HttpCodes } from '../lib/structures/http/HttpCodes';
+import type { MediaParserStore } from '../lib/structures/MediaParserStore';
 import { Middleware } from '../lib/structures/Middleware';
 import type { Route } from '../lib/structures/Route';
-import { MimeTypes } from '../lib/utils/MimeTypes';
 
 export class PluginMiddleware extends Middleware {
 	private readonly maximumBodyLength: number;
+	private readonly mediaParsers: MediaParserStore;
 	public constructor(context: PieceContext) {
 		super(context, { position: 20 });
 		this.maximumBodyLength = this.context.server.options.maximumBodyLength ?? 1024 * 1024 * 50;
```

Comment:
```suggestion
		this.maximumBodyLength = this.context.server.options.maximumBodyLength ?? 52428800; // 50Mb
```
Would be more computationally efficient to declare a static value rather than making the process run the calculation every time this is constructed. Very small nitpicky efficiency gain, but a gain nonetheless.

Reply:
> Would be more computationally efficient to declare a static value rather than making the process run the calculation every time this is constructed.

And... how many times is this constructed?